compose/config/schema.json

```diff
@@ -62,9 +62,9 @@
           "type": "object",
 
           "properties": {
-            "address": {"type": "string"}
+            "syslog-address": {"type": "string"}
           },
-          "required": ["address"]
+          "required": ["syslog-address"]
```

Comment:
Unless I've misunderstood, we shouldn't be validating properties in here at all - we don't know what options the driver will accept.